src/formatting/vertical.rs

```diff
@@ -50,7 +50,7 @@ impl AlignedItem for ast::StructField {
             mk_sp(self.attrs.last().unwrap().span.hi(), self.span.lo())
         };
         let attrs_extendable = self.ident.is_none() && is_attributes_extendable(&attrs_str);
-        rewrite_struct_field_prefix(context, self).and_then(|field_str| {
+        Some(rewrite_struct_field_prefix(context, self)).and_then(|field_str| {
```

Comment:
This looks a bit odd. Could we tweak a bit, maybe just create a local with the field str or move the call down below where the corresponding arg is being passed?

Reply:
Yeah, ideally, clippy would catch `Some(x).and_then()` :)

Reply:
That would be helpful indeed!